benchbuild/source/http.py

```diff
@@ -0,0 +1,75 @@
+"""
+Declare a http source.
+"""
+import typing as tp
+
+import attr
+import plumbum as pb
+from plumbum import local
```

Comment:
Codacy found an issue: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)